exchange/exchange.go

```diff
@@ -497,6 +508,30 @@ func (e *exchange) getAllBids(
 	return adapterBids, adapterExtra, bidsFound
 }
 
+func insertStoredImpData(impsToStoredRequest map[string][]byte, bids *pbsOrtbSeatBid) error {
+	for _, bid := range bids.bids {
+		if val, ok := impsToStoredRequest[bid.bid.ImpID]; ok {
+			bidExtData := make(map[string]interface{})
+			storedImpVal := make(map[string]interface{})
+			err := json.Unmarshal(bid.bid.Ext, &bidExtData)
+			if err != nil {
+				return err
+			}
+			err = json.Unmarshal(val, &storedImpVal)
+			if err != nil {
+				return err
+			}
+			bidExtData[StoredRequestAttributes] = storedImpVal
```

Comment:
Is there a reason why we are inserting the entire stored request object here? I see the ask in the spec is to parse and include only video attributes.
```
Prebid Server will be required to parse the Stored Request object to look for video attributes. If one or more attributes are present, echo in OpenRTB extension object in the response. Otherwise, do not include the "storedRequestAttributes" object.
```

I check Prebid Java and they parse the video attributes of the stored imp and echo just those in the bid ext as well, unless I am missing something. Check here: https://github.com/prebid/prebid-server-java/pull/537/files#diff-f0a1b91c0eb7400d0051d08a2f6b06bedd05e1462b33501abe80cd8641439334R140. 

Also, this is probably too late as this has already been implemented in Java but having the flag specific to video attributes and the bid response ext being more general `storedrequestattributes` is kinda confusing :/

Reply:
Good point! Modified to add stored request attributes for video imps only. Also added another check if stored imp had a video data itself (it may have only `ext `for example). If there is video data in stored impression - then I put it back to `bid.ext. storedrequestattributes`